src/util/Conversions.h

```diff
@@ -246,8 +265,9 @@ string convertFloatToIndexWord(const string& orig, size_t nofExponentDigits,
   }
 
   // Add padding to the exponent;
-  AD_CHECK_GT(nofExponentDigits, expoString.size());
-  size_t nofPaddingElems = nofExponentDigits - expoString.size();
+  AD_CHECK_GT(DEFAULT_NOF_VALUE_EXPONENT_DIGITS, expoString.size());
```

Comment:
Given that we no longer override its value we could remove the `DEFAULT` from `DEFAULT_NOF_VALUE_EXPONENT_DIGITS`